src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md

```diff
@@ -0,0 +1,93 @@
+# GraphQL
+
+## Queries
+
+Queries are made up of three areas, the `type`, `arguments` and `return values`, an example would be;
+
+```json
+query {  
+    blog (contentItemId:'123')
+    {
+        ContentType
+    }
+}
+```
+
+Where `blog` is the type, `contentItemId: '123'` is the argument, and `ContentType` is the return value
+
+In Orchard we allow you to define whatever you want to return, however this must be declared upfront.
+
+### Define a type
+
+You can define a type in two ways, through either DI registration or Auto Discovery
+
+#### DI Registration.
+
+All query types registered in DI must inherit off the class `QueryFieldType`. 
```

Comment:
Does this README need to be reviewed? I can't find any references to some of the mentioned classes e.g. QueryFieldType, IQueryFieldTypeProvider. These got changed to ISchemaBuilder?

This PR is looking great btw!

Reply:
Ah - yeah, ill look at that tomorrow. 

Can you pull this branch down and run the tests locally???? I just want to see if the CI boxes are screwed or this PR is